test/test_sparse.py

```diff
@@ -2177,6 +2177,20 @@ def test_factory_empty_indices(self):
         expected_indices = torch.empty((4, 0), dtype=torch.long, device=device)
         self.assertEqual(tensor._indices(), expected_indices)
 
+    def test_sparse_abs(self):
```

Comment:
As mentioned (here)[https://github.com/pytorch/pytorch/issues/46761#issuecomment-716511793], this test really highlights the importance of us investing in our sparse testing infrastructure. We should have automatically generated tests for unary sparse ops that use our latest test tools. This would make writing these tests trivial.